src/components/Tasks/Toolbar.tsx

```diff
@@ -49,11 +96,40 @@ export default function Toolbar(props: PropsType) {
   });
 
   return (
-    <div style={{ display: 'flex', justifyContent: 'space-between' }}>
-      {defaultCollection && <QuickAdd style={{ flexGrow: 1, marginRight: '0.75em' }} onSubmit={onItemSave} defaultCollection={defaultCollection} />}
+    <div style={{ display: 'flex', justifyContent: 'flex-end', alignItems: 'center' }}>
+      <Transition in={showSearchField} timeout={transitionTimeout}>
+        {(state) => (
+          <TextField
+            fullWidth
+            placeholder="Search"
+            value={searchTerm}
+            color="secondary"
+            margin="dense"
+            variant="filled"
+            className={classes.autocomplete}
+            style={transitionStyles[state]}
+            onChange={(e) => setSearchTerm(e.target.value)}
+            InputProps={{
+              className: classes.input,
+              startAdornment: (
+                <InputAdornment position="start" className={classes.adornment}>
+                  <SearchIcon />
+                </InputAdornment>
+              ),
+            }}
+          />
+        )}
+      </Transition>
+
+      <div className={classes.button}>
+        <IconButton size="small" onClick={toggleSearchField}>
```

Comment:
This icon, like all the rest, are missing the `title` prop. Look at the refresh icon at the top right how it says "Refresh" if you hover with the mouse. This is not the case in your toolbar icons.